vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -123,7 +123,7 @@ function InstallVS {
 function CheckNode {
     try {
         $v = (Get-Command node -ErrorAction Stop).Version.Major
-        return $v -eq 12 -or $v -eq 13 -or $v -eq 14
+        return ($v -ge 12) -and (($v % 2) -eq 0);
```

Comment:
shouldn't this just plain-out check for 16 now then?

Reply:
So far we've claimed to still support Node 12. We transpile down to that level, but we use Node 14 typings, so we could potentially add API calls that didn't work on older versions... I'm not aware that we have any known issues with 12 though (where I do remember us no longer supporting 10).

Bumping the min supported to Node 14 seems like something we might want to do at some point though. I think RN itself just dropped Node 10 though, so it might be a little early.

The %2 bit is because Node has every other version be an LTS. The script installs latest LTS, so it can be a bit confusing to have it say install 16, or later 18, then the check fails what was just installed. This makes the check permissive enough so that anything it installs should be reported as acceptable, but to flag people installing non-LTS versions which we may not have tested.